tools/c7n_azure/c7n_azure/resources/cosmos_db.py

```diff
@@ -88,6 +88,15 @@ class resource_type(ChildTypeInfo):
         raise_on_exception = False
         annotate_parent = True
 
+        id = '_rid'
+        name = 'id'
```

Comment:
These two can go away now

Reply:
why?

Reply:
I just tried commenting them out, and `name` can be removed, but `id` cannot be. The issue comes from line 155 of `/c7n/reports/csvout.py` where it references the id field of the resource_type

Reply:
Oh ok.  But should name be there?  Looks like that is just the higher level default report field but once we specify them then that doesn't get used?

Reply:
Ah, I guess we should have had one of these :)  
https://github.com/cloud-custodian/cloud-custodian/blob/master/tools/c7n_gcp/tests/test_resource.py#L21

Reply:
That would be a good test to have. Tomorrow I will work on implementing it in the c7n_azure tests and will remove the name field

Reply:
We can move `name = 'id'` to the base TypeInfo since every resource needs it

Reply:
Where does a resource need this field? It might but so far I am not aware it is needed at all so I want to confirm :)

Reply:
report needs this field